### PR TITLE
Change API's concurrent default option to skip

### PR DESCRIPTION
**Pull request by kunkun-tang**
Today, given a schedule, the default option for concurrent setting is `ignore`, that is, allowing concurrent executions whatever schedule triggers. This could be regarded as a bug in current AZ. The reason is that, In UI, the default schedule option is "skip" while API's default option is "ignore". The inconsistency between API and UI should be resolved.

This commit fixes the option to `skip`. When next schedule triggers, as long as the last execution hasn't finished, it won't trigger the next execution. As a consequence, this commit changes API's default option to "skip" and UI's default behavior is not affected. Tested on our staging cluster, and working.

### CODE REVIEW

File: azkaban-common/src/main/java/azkaban/server/HttpRequestUtils.java [hunk 1]
```diff
@@ -87,16 +87,17 @@ public static ExecutionOptions parseFlowOptions(final HttpServletRequest req)
     String concurrentOption = "skip";
```
Comment (reallocf) [inline azkaban-common/src/main/java/azkaban/server/HttpRequestUtils.java]: Can you just say:
`String concurrentOption = getParam(req, "concurrentOption", "skip");`
Or something like that so we don't need the if statement at all?

Comment (kunkun-tang) [inline azkaban-common/src/main/java/azkaban/server/HttpRequestUtils.java]: You are right!

**Comment by HappyRay** [comment]
Please explain in the commit message the motivation for this change.

My understanding is that this is to make the API behavior consistent with the existing UI.

**Comment by kunkun-tang** [comment]
PR was updated. @HappyRay @reallocf

**Comment by HappyRay** [comment]
Thanks
Could you make it clear in the commit summary line that this change applies to the API?

**Comment by kunkun-tang** [comment]
Thanks for your review. updated the description. @HappyRay

**Pull request merged**